nilearn/interfaces/fmriprep/load_confounds.py

```diff
@@ -57,10 +61,10 @@ def _check_error(missing):
     """Consolidate a single error message across multiple missing confounds."""
     if missing["confounds"] or missing["keywords"]:
         error_msg = (
-            "The following keywords or parameters are missing: "
-            + f" {missing['confounds']}"
-            + f" {missing['keywords']}"
-            + ". You may want to try a different denoising strategy."
+                "The following keywords or parameters are missing: "
```

Comment:
I think you should remove the added indentation for these lines to make PEP8 happy